test-plone-5.1.x.cfg

```diff
@@ -0,0 +1,19 @@
+[buildout]
+extends =
+    https://raw.github.com/4teamwork/ftw-buildouts/master/test-plone-5.1.x.cfg
+    sources.cfg
+
+package-name = ftw.publisher.core
+
+auto-checkout =
+    plonetheme.onegov
+    ftw.simplelayout
+
+
+[branches]
+plonetheme.onegov = plone5
+ftw.simplelayout = plone-5.1.x
```

Comment:
Are we going to wait for the simplelayout Plone 5 support to be merged?

Reply:
It needs the new plone 5 simplelayout for the tests to run.
Otheriwse it gets:

Reply:
I just asked @maethu about this and the problem is, that `ftw.simplelayout` has a dependency on `plonetheme.blueberry` which in turn depends on `ftw.publisher.core` (egg/hen problem). 

We need to leave this as is.

Reply:
@jone, should we merge this?

Reply:
@mbaechtold yes, please merge it 👍